ledger/ledger-common/src/main/kotlin/net/corda/v5/ledger/common/transactions/LedgerTransaction.kt

```diff
@@ -0,0 +1,16 @@
+package net.corda.v5.ledger.common.transactions
+
+import net.corda.v5.base.annotations.CordaSerializable
+import net.corda.v5.base.annotations.DoNotImplement
+import net.corda.v5.crypto.SecureHash
+import java.security.PublicKey
+
+@DoNotImplement
+@CordaSerializable // TODO(???)
+interface LedgerTransaction {
+    val id: SecureHash
+    val privacySalt: PrivacySalt
```

Comment:
Why do we have a `privacySalt` here but also within `wireTransaction`? Are they the same?

Reply:
Yes. I was not too sure about it. I have just removed it for the time being.